apps/rush/package.json

```diff
@@ -41,6 +41,8 @@
     "@rushstack/eslint-config": "workspace:*",
```

Comment:
Should we add a "start" script to package.json?

Reply:
To run `lib/start.js` or `lib/start-dev.js`? I usually don't test Rush inside the repo I'm developing in - I usually use another clone of Rushstack or a different repo entirely.